shopfloor/models/stock_move.py

```diff
@@ -36,3 +36,36 @@ def _action_done(self, cancel_backorder=False):
                 if picking.state == "done":
                     picking.action_done()
         return moves
+
+    def action_done_in_picking(self):
```

Comment:
Looking for a better name...

Reply:
Shouldn't we pass the moves or move lines that we want to set to extract and set to done?

Reply:
If so, `extract_moves_and_action_done` or `extract_move_lines_and_action_done`?

Reply:
> Shouldn't we pass the moves or move lines that we want to set to extract and set to done?

Because in some cases, we deliver the lines one by one in a different location

Reply:
Yes, but I supposed that for this purpose we are using the other method `split_other_move_lines` (as it is already done in some scenarios), and from the resulting moves we put them in a separate picking with the current method. Am I wrong?

Reply:
But assuming you split the move lines in a new move already, this method still needs to know which move of the picking you want to extract.

Reply:
the move(s) to extract is `self` here